README.md

```diff
@@ -157,3 +159,12 @@ Compile and run tests:
 Tests would normally run in many JVM configurations. If you are contributing the code,
 please try to run the build on multiple JDK releases, most importantly 8u and 11u.
 GitHub workflow "JOL Pre-Integration Tests" should pass with your changes.
+
+## Related projects
+
+* [IntelliJ IDEA JOL Plugin](https://github.com/stokito/IdeaJol) can estimate object size and has an inspection to find heavy classes
+* Heap dump `*.hprof` files analysers:
```

Comment:
Heap dump decoding is very experimental in JOL, and hardly a target for current development. So listing heap dump analyzers looks excessive.

Reply:
Agree, still this is something that can be used to solve memory issues. For newcomers this may be time saver to see such related projects. If you think that this is excessive I'll remove the commit or left IdeaJOL only (if you don't mind)

Reply:
Yeah, mention IdeaJOL, don't mention hprof analyzers.